src/modules/navigator/mission_params.c

```diff
@@ -156,3 +156,17 @@ PARAM_DEFINE_FLOAT(MIS_YAW_TMT, -1.0f);
  * @group Mission
  */
 PARAM_DEFINE_FLOAT(MIS_YAW_ERR, 12.0f);
+
+/**
+ * Allow waypoints below home altitude.
+ *
+ * If enabled, mission will not be validated for waypoints below home altitude,
```

Comment:
To me, this sounds like all validation checks will be skipped for waypoints that are below home altitude. Not a big deal, but can be confusing

Reply:
updated to be a bit more clear